static/build/.tmp_update/runtime.sh

```diff
@@ -102,20 +102,29 @@ main() {
         check_main_ui
 
         check_networking  		
-              
+        ntp_updater
+		
```

Comment:
Couldn't this just be in `check_networking` and not clutter the main loop?

Reply:
You'd think so but when I had it in there the change was always 1 cycle behind, for example:

Boot system, NTP set time to UTC (as no TZ is set) (00:00) even though UTC-1 is selected (01:00)
Enter tweaks - TZ changed to UTC-2, exit tweaks, time now says 01:00 (should be 02:00)
Enter tweaks - TZ changed to UTC-1, exit tweaks, time now says 02:00 (should be 01:00)
Enter tweaks - TZ changed to UTC-2, exit tweaks, time now says 01:00 (should be 02:00)
Enter tweaks - TZ changed to UTC-1, exit tweaks, time now says 02:00 (should be 01:00)

Even though it's the exact same thing in practise whether its in check_net or in the loop itself... Let me know if you have the same issue with it in check_net and I'll see if I can fix it.